src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java

```diff
@@ -44,31 +45,33 @@ public VerificationResult verifyZip(String blobName, byte[] rawBlob) {
             return ok();
         } catch (DocSignatureFailureException ex) {
             logger.info("Invalid signature. Blob name: {}", blobName, ex);
-            return error("Invalid signature");
+            return error(ErrorCode.ERR_SIG_VERIFY_FAILED, "Invalid signature");
         } catch (InvalidZipArchiveException ex) {
             logger.info("Invalid zip archive. Blob name: {}", blobName, ex);
-            return error("Invalid zip archive");
+            return error(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
         } catch (IOException ex) {
             logger.info("Error occurred when verifying file. Blob name: {}", blobName, ex);
-            return error(null);
+            return error(null, null);
```

Comment:
is the code not required?

Reply:
what should be the value? ERR_ZIP_PROCESSING_FAILED sounds good?